src/components/Issuance/EmailIssuance.tsx

```diff
@@ -127,11 +131,10 @@ const EmailIssuance = () => {
 						}
 						setLoading(false);
 				}
-			    
-
-			  else  if (currentSchemaType === SchemaTypes.schema_W3C && orgId && allSchemaSelectedFlag === 'true') {
-					const response = await getAllSchemas(schemaListAPIParameter,currentSchemaType); 
-					const { data } = response as AxiosResponse;
+			    	
+			  else if (currentSchemaType === SchemaTypes.schema_W3C && orgId && allSchemaSelectedFlag) {
+					const  response = await getAllSchemas(schemaListAPIParameter,currentSchemaType); 
+					const { data } = response as unknown as AxiosResponse;
```

Comment:
Any specific reason for using unknown type?

Reply:
Resolved.

Reply:
Resolved.